lib/twilio-ruby/rest/accounts/v1/credential/aws.rb

```diff
@@ -254,7 +254,7 @@ def initialize(version, payload, sid: nil)
               @properties = {
                   'sid' => payload['sid'],
                   'account_sid' => payload['account_sid'],
-                  'friendly_name' => payload['friendly_name'],
+                  'friendly_name' => payload['friendly_name'].to_json,
```

Comment:
This is during deserialization of the API response. Shouldn't the fix be during serialization to generate the request payload?